tasks/db_deploy/test/manual_tests/README.md

```diff
@@ -882,6 +889,4 @@ You are now connected to database "postgres" as user "postgres".
 DROP DATABASE
 
 postgres=# \q
-```
-
-
+```
```

Comment:
Cleanup for no migration will have additional output around line 878. When the SCHEMA is dropped, your tables will be dropped too. You will see somenting like 

```
drop cascades to table orca.files
drop cascades to table orca.granules
drop cascades to table orca.provider_collections_xref
drop cascades to table orca.collections
drop cascades to table orca.providers
```

Reply:
added

Reply:
Per our discussion, those should be added around line 890 without the additional items added